plugin.video.mytv_bg/addon.py

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#imports
+from xbmcswift import Plugin, xbmc, xbmcaddon, xbmcplugin, xbmcgui, clean_dict
+from urllib2 import urlopen, HTTPError
+import xbmcgui
+from pyxbmct.addonwindow import *
+import sys
+import json
+import os.path
+from urllib2 import Request
+import urllib
+import urllib2
+
+__addon_name__		= xbmcaddon.Addon().getAddonInfo('name')
+__id__				= xbmcaddon.Addon().getAddonInfo('id')
+__lang__			= xbmcaddon.Addon().getLocalizedString
+__addon__ 			= xbmcaddon.Addon()
+__version__ 		= __addon__.getAddonInfo('version')
+__profile_path__ 	= xbmc.translatePath( __addon__.getAddonInfo('profile') ).decode("utf-8")
+__token_filepath__	= __profile_path__ + '/token.txt'
+
+
+THUMBNAIL_VIEW_IDS = {'skin.confluence': 500,
```

Comment:
This is not allowed anymore in the repository I am affraid. Dev's should set the Content Type of the items correctly and not overwrite the default skin views